changelog_unreleased/typescript/14701.md

```diff
@@ -0,0 +1,52 @@
+#### Keep parentheses around `TSInstantiationExpression` followed by a property access (#14701 by @morsko1)
+
+<!-- prettier-ignore -->
+```ts
+// Input
+(Array<string>).name;
+(fn1<string>).bind(obj);
+(fn2<string, number>).bind(obj);
+a[(Array<string>)];
+a[(Array<string>).name];
+(Array<string>).a;
+(Array<string>)?.a;
+(Array<string>)[a];
+(Array<string>)?.[a];
+(Array<string>)["a"];
+(Array<string>)?.["a"];
+(Array<string>)[`a`];
+(Array<string>)?.[`a`];
+(Array<string>)[(Array<string>)];
```

Comment:
We don't need put everything in changelog, I think the following cases are enough.

```js
(Array<string>).a;
(Array<string>)?.a;
(Array<string>)[a];
(Array<string>)?.[a];
```

Reply:
updated